requirements.txt

```diff
@@ -9,9 +9,3 @@ odfpy==1.3.3
 git+https://github.com/alphagov/digitalmarketplace-utils.git@26.2.1#egg=digitalmarketplace-utils==26.2.1
 git+https://github.com/alphagov/digitalmarketplace-content-loader.git@4.4.1#egg=digitalmarketplace-content-loader==4.4.1
 git+https://github.com/alphagov/digitalmarketplace-apiclient.git@8.8.0#egg=digitalmarketplace-apiclient==8.8.0
-
```

Comment:
Yeah, we can drop it everywhere. We don't need these dependency for Docker deployments.